portfolio/src/main/webapp/script.js

```diff
@@ -52,8 +52,27 @@ function makeUglySite() {
   toggleVisibility('hide-div');
 }
 
-function getComments() {
-  const url = '/data?limit=' + getLimit();
+function setLanguage() {
+  let input = document.getElementById("language");
+  let languageCode = input.options[input.selectedIndex].value;
+  const params = new URLSearchParams({'/data?language': languageCode});
+  getComments(params);
+}
+
+function setParams(parameters = null) {
```

Comment:
instead of defaulting to null, you could default to `(parameters = new URLSearchParams())`

That way, you don't need the if/else, you can always call set